### PR TITLE
Strengthen OpenID provider language & list OpenID providers

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: docs/security/user-certs.md [hunk 1]
```diff
@@ -34,15 +34,13 @@ Follow the steps below to get an user certificate:
...
-
     ![Institution Selection](/img/cilogon_select_idp.png).
 
-    !!! note
-        Make sure to select your educational institution from CILogon's list.
-        For instance, do not use OpenID (e.g. Google, Github, etc.) as a provider since not all OSG resources support
-        certificates using an OpenID provider.
+    !!! warning
+        Do not use Google, GitHub, or ORCID as providers, because they use the OpenID CA and many OSG resources
+        do not support certificates using that CA.
```
Comment (brianhlin) [inline docs/security/user-certs.md]: I'd reword this to something like:

> Do not use Google, GitHub, or ORCID as providers since they are not widely supported in the OSG.

Comment (matyasselmeci) [reply docs/security/user-certs.md]: OK
